apps/system/test/unit/settings_migrator_test.js

```diff
@@ -73,4 +79,25 @@ suite('system/settings_migrator', function() {
       assert.equal(window.navigator.mozSettings.mSettings[keyHour12], true);
     });
   });
+
+
+  suite('when old passcode is present', function() {
```

Comment:
expect to see `if 'lockscreen.passcode-lock.digest.value' exists` and `if 'lockscreen.passcode-lock.digest.value' value not exist` test cases. The test does not mean the migration succeed